script.js

```diff
@@ -0,0 +1,13 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n
+  .addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    navMenu.classList.remove('active');
+  }));
```

Comment:
- [ ] You have done great so far. :+1: But there are still some design requirements you need to meet before moving to the next milestone.
- - As part of the requirement for this milestone is to try to replicate the Figma design as much as you can, Your mobile menu doesn't fully match the Figma design, looking at the screenshot below, you can see that according to the Figma design, when the menu is open, its meant to cover the full screen, but that's not the case in your own project. I'll suggest you update this :smile: :wave: 

- - FIGMA DESIGN:
![Screenshot from 2023-06-29 13-15-37](https://github.com/ishimwepati/Portfolio/assets/107632780/5006981e-9248-4950-ba9e-4b83c68868ce)

- - YOUR DESIGN:
![Selection_070](https://github.com/ishimwepati/Portfolio/assets/107632780/96ea21f2-e071-489e-9c03-06565c86209d)

 - - Also the color of the close icon doesn't fully match the Figma design, I'll suggest you update this too. :smile: